druid-shell/src/platform/windows/mod.rs

```diff
@@ -26,3 +26,194 @@ pub mod runloop;
 mod timers;
 pub mod util;
 pub mod window;
+
+// from winapi::um::d2d1;
+// need to move from one type to another?
+//
+// https://docs.microsoft.com/en-us/windows/win32/direct2d/render-targets-overview
+// ID2D1RenderTarget is the interface. The other resources inherit from it.
+//
+// A Render Target creates resources for drawing and performs drawing operations.
+//
+// - ID2D1HwndRenderTarget objects render content to a window.
+// - ID2D1DCRenderTarget objects render to a GDI device context.
+// - bitmap render target objects render to off-screen bitmap.
+// - DXGI render target objects render to  a DXGI surface for use with Direct3D.
+//
+// https://docs.microsoft.com/en-us/windows/win32/direct2d/devices-and-device-contexts
+// A Device Context, ID2D1DeviceContext, is used for windows 8 and higher. Render Target
```

Comment:
This might be a misunderstanding. DeviceContext is available as of Windows 7  with platform update, which the minimum compatibility target for druid. We are not making an effort do to RenderTarget only (ie if the downcast to DC fails).